frontend/src/App.test.tsx

```diff
@@ -360,13 +360,13 @@ describe("App.handleNewReport", () => {
       },
       sessionState: {
         runOnSave: false,
-        reportIsRunning: false,
+        scriptIsRunning: false,
       },
       sessionId: "sessionId",
       commandLine: "commandLine",
     },
   }
-  const NEW_REPORT = new NewReport(NEW_REPORT_JSON)
+  const NEW_REPORT = new NewSession(NEW_REPORT_JSON)
```

Comment:
nit: looks like the `NEW_REPORT` and `NEW_REPORT_JSON` consts snuck by without being renamed